xline/src/server/lease_server.rs

```diff
@@ -74,12 +74,12 @@ impl Lease for LeaseServer {
         debug!("Receive LeaseKeepAliveRequest {:?}", request);
         // TODO: Temporary solution
         let mut req_stream = request.into_inner();
-        let (tx, rx) = mpsc::channel(100);
+        let (tx, rx) = mpsc::channel(128);
```

Comment:
how do we determine the channel buffer size?

Reply:
This is the experience from etcd. We may need to tune this in the future. 
Let me define a const first.